locales/en/translation.json

```diff
@@ -0,0 +1,176 @@
+{
+  "Tidepool Uploader":"",
```

Comment:
Shouldn't we be using the same for key and value everywhere in this file, to prevent everything showing up in `locales/en/translation.missing.json` again?